### ISSUE TITLE
Keeps adding new line before block comment

### ISSUE THREAD

**Issue by Andikki** [tierratelematics/code-beautifier#11]
If you set preserve_newlines in options, it keeps adding a new line before a comment block, if this comment block is between rules.

settings.json:
```json
"beautify.options": {
    "preserve_newlines": true
},
```

Initial css:
```css
.some-rule {
  width: 20px;
}

/* Description for the next rules */
.other-rule {
  /* width: 40%; */
  width: 20px;
}

```

After a few beautifies:
```css
.some-rule {
  width: 20px;
}







/* Description for the next rules */

.other-rule {
  /* width: 40%; */
  width: 20px;
}
```

**Comment by jasamour** [comment]
Having the same issue. I've tried using other beautify settings to correct this issue but to no avail.

**Not Working:**
```
   "max_preserve_newlines": 5,
/* This would at least stop the new line chain from expanding further */

   "unformatted": "comments",
/* It's listed in the JS Beautifier url poseted in this extension. Had to try */

   "newline-between-rules": false,
/* It's listed in the JS Beautifier url poseted in this extension. Had to try */

   "css.format.extraLiners": null,
/* It's listed in the JS Beautifier url poseted in this extension. Had to try */
```

**Something in the code of this extension is automatically adding new lines before a comment block and no extensions settings can be added to counteract this bug.**

**Comment by mike7515** [comment]
Fixed in 2.0.3
